### DISCUSSION TITLE
v1.1.30

### DISCUSSION THREAD

**Discussion by javamonn** [javamonn/literal#132] [Announcements]
# [v1.1.30] - 2021-08-28

Adds annotation export (text and markdown) and includes source path name when viewing a collection.

### Added
- Adds functionality to export individual annotations in text or markdown format via the Android Share sheet.

### Changed
- When viewing a source annotation collection the pathname of the source is displayed within the header.
- Removes extraneous console logs displaying analytics instrumentation.

<hr /><em>This discussion was created from the release <a href='https://github.com/literal-io/literal/releases/tag/'>v1.1.30</a>.</em>

**Comment by sskingyungjp2000** [comment]
This Shit Go HARD!!!!
